homeassistant/components/risco/config_flow.py

```diff
@@ -151,7 +152,7 @@ async def async_step_local(self, user_input=None):
                 self._abort_if_unique_id_configured()
 
                 # Risco can hang if we don't wait before creating a new connection
-                await asyncio.sleep(SLEEP_INTERVAL)
```

Comment:
Is there a reason we can't raise ConfigEntryNotReady and try again later instead?

If not it would be better to set this to a higher reasonable value as asking the user to know how long to sleep is not likely something they are going to know

Reply:
Thanks. I see your point, but it's going to take me a while to test adding a timeout and raising `ConfigEntryNotReady`, and how Risco behaves with it, and I won't be ready in time for the beta.
I can hopefully get it done over the weekend, and perhaps it'll be considered post-beta since this is a bug fix, but maybe in the mean time I just increase the default sleep to 5 instead of this PR? This seems like a safe value from my testing.

Reply:
As long as the resulting change is small, targeted, and clearly a bug fix, we can consider it for beta or in a patch.